sys-apps/config-site/files/config.site.in

```diff
@@ -0,0 +1,10 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+for script in "@datarootdir@/config.site.d"/*.conf; do
+	test -f "${script}" || continue
+	test -r "${script}" || continue
```

Comment:
I wonder if we should emit a warning here, to avoid confusing users why some files are skipped.

Reply:
configure silently skips site files if they are not readable (see above snippet).